boxoffice/templates/cash_receipt_new.html.jinja2

```diff
@@ -0,0 +1,557 @@
+{%- extends "layout_email.html.jinja2" -%}
+{%- from "macros_email.html.jinja2" import cta_button -%}
+{%- block content -%}
+  <br/>
+  <hr/>
+  <tr align="right">
+    <td class="receipt-header" colspan="4">
+      <p>{%- trans %}Receipt no. {% endtrans -%}<span>{{ order.receipt_no }}</span></p>
+    </td>
+  </tr>
+  <tr align="left">
+    <td class="invoice-header" colspan="4">
+      <p>{%- trans %}Hasgeek Learning Private Limited {% endtrans -%}</p>
+    </td>
+  </tr>
+  <tr>
+    <td style="width: 100%;" align="left" valign="top" colspan="4">
+      <!--[if (gte mso 9)|(IE)]>
+      <table cellpadding="0" cellspacing="0" border="0" role="presentation" style="width: 100%;">
+      <tr>
+      <td valign="top" style="width: 49.5%;">
+      <![endif]-->
+        <div style="display:inline-block;vertical-align:top;width: auto; margin-right: 68px">
+          <table>
+            <tr align="left">
+              <td class="invoice-label">
+                <p>{%- trans %}Address {% endtrans -%}</p>
+                {%- if org.details.get('address') %}
```

Comment:
Move these `if` conditions out, to wrap the entire `<tr>`. Else we'll have a label with a blank value. Same for the PAN/CIN rows below.